### PR TITLE
docs(link-button): add link button topic #194

### LINKED ISSUE #194: Add a topic for Link Button component

### PR DESCRIPTION
**Pull request by zhosti**
Closes #194

### CODE REVIEW

File: doc/en/components/link-button.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+<igc-link-button variant="contained" disabled="true">Disabled</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-disabled">
+</iframe></div>
+
+### Download
+
+By setting a URL to `download` property, we can make the link button to download file when is clicked.
```
Comment (gedinakova) [inline doc/en/components/link-button.md:121]:
```suggestion
By setting a URL to `download` property, we can make the link button download a file when it is clicked.
```

**Review by gedinakova — CHANGES_REQUESTED** [review]

File: doc/en/components/link-button.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+<igc-link-button
+    variant="flat"
+    href="https://www.infragistics.com/webcomponentssite/components/general-getting-started.html">
+    Flat
+</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-flat">
+</iframe></div>
```
Comment (DiyanDimitrov) [inline doc/en/components/link-button.md:57]: Shouldn't we use `<code-view>` instead?

File: doc/en/components/link-button.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-fab">
+</iframe></div>
+
+### Disable Link Button
+
+The `disabled` property can be used to make a button unclickable:
+
+```html
+<igc-link-button variant="contained" disabled="true">Disabled</igc-link-button>
```
Comment (DiyanDimitrov) [inline doc/en/components/link-button.md:112]: The correct way for setting the `disabled` attribute.
```suggestion
<igc-link-button variant="contained" disabled>Disabled</igc-link-button>
```

File: doc/en/components/link-button.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+    target="_blank">
+    Download
+</igc-link-button>
+```
+
+<div class="sample-container loading" style="height: 70px">
+    <iframe class="lazyload" seamless width="100%" height="100%" frameborder="0" data-src="{environment:dvDemosBaseUrl}/inputs/link-button-download">
+</iframe></div>
+
+## Link Button Sizing
```
Comment (DiyanDimitrov) [inline doc/en/components/link-button.md:136]: I am not sure if we should repeat the name of the component in the sub-headers. Maybe just 'Size' is enough.

File: doc/en/components/link-button.md (hunk 2 above)
Comment (zhosti) [reply doc/en/components/link-button.md:57]: I follow this approach from Angular button topic.

**Review by DiyanDimitrov — APPROVED** [review]

**Review by gedinakova — APPROVED** [review]

**Review by gedinakova — APPROVED** [review]

**Pull request merged**